<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Manage</string>
  </property>
  <widget class="QPushButton" name="input_bt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>input Folder</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_rt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="output_bt">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>90</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>output Folder</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="output_rt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="process_bt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_bt">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Processing</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
